src/containers/pages/JurisdictionAssignment/JurisdictionCell/index.tsx

```diff
@@ -33,7 +33,12 @@ const NodeCell = (props: NodeCellProps) => {
   }
 
   return (
-    <Link to={`${baseUrl}/${node.model.id}`} key={`${node.model.id}-span`}>
+    <Link
+      to={`${baseUrl}/${node.model.id}`}
+      key={`${node.model.id}-span`}
+      // tslint:disable-next-line: jsx-no-lambda
+      onClick={() => fetchUpdatedCurrentParent('', false)}
```

Comment:
Why does this have to happen as an onClick event to `Link`?  Can we find another way please?